src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Navbar from "../../components/Navbar/Navbar";
+import Hero from "../../components/Hero/Hero";
+
+function Home() {
+  return (
+    <>
```

Comment:
Instead of using a fragment `<>...</>`, always try to use a regular div and give it a meaningful name such as:
```tsx
<div className='home-container'>
```
This will help you avoid css specificity issues.